### DISCUSSION TITLE
Git clones

### DISCUSSION THREAD

**Discussion by Pr0gramm3r2022** [community/community#131538] [Code Security]
### Select Topic Area

General

### Body

How can I see who cloned my GitHub Repository. I can very easily see how many clones there are of my repo because the graphs clearly show that, but I want to know specifically who cloned my repository.

**Comment by iamdulanga** [comment]
Hello, @Pr0gramm3r2022 

GitHub does not provide built-in features to directly track who cloned your repository. Cloning a repository is typically an anonymous operation and GitHub's focus is on facilitating collaboration rather than monitoring individual actions like cloning. If you want to track who accesses or uses your code, you might consider implementing analytics or tracking mechanisms within your application or repository, though this would generally require custom solutions outside of GitHub's platform
